src/library/DIPS.Mobile.UI/Components/TextFields/InputFields/MultiLineInputField/MultiLineInputField.cs

```diff
@@ -79,21 +92,28 @@ private void CreateButtons()
             Command = new Command(OnCancelTapped)
         };
 
-        var doneButton = new Button
+        m_doneButton = new Button
         {
             Text = DUILocalizedStrings.Save, 
             Style = Styles.GetButtonStyle(ButtonStyle.PrimarySmall),
             Command = new Command(OnSaveTapped),
             CommandParameter = InputView.Text
         };
-
-        m_buttonsLayout = new HorizontalStackLayout 
+        
+        m_buttonsLayout = new Grid
         { 
-            HorizontalOptions = LayoutOptions.End,
-            Spacing = 10,
-            Margin = new Thickness(0, 8, 0, 0),
-            Children = { cancelButton, doneButton } 
+            ColumnDefinitions =
+            [
+                new ColumnDefinition { Width = GridLength.Star },
```

Comment:
Har du testet med litt større skriftstørrelse? for å se at oppførselen er grei om man har litt dårlig plass.

Reply:
Kjører truncation og maks 1 linje, tenker det må være innafor?

![image](https://github.com/user-attachments/assets/c106935e-f062-4abc-a853-66b554ea2cde)